text/????-separate-semantic-conventions.md

```diff
@@ -0,0 +1,115 @@
+# Separate Semantic Conventions
+
+Move Semantic Conventions outside of the main Specificaiton and version them
+separately.
+
+## Motivation
+
+We need to allow semantic conventions to evolve independently of the overall
+OpenTelemetry specification. Today, any breaking change in a semantic convention
+would require bumping the version number of the entirety of the OpenTelemetry
+specification.
+
+## Explanation
+
+A new github repository called `semantic-conventions` would be created in the
```

Comment:
Since you stressed recently that some compatibility requirements for instrumentations are not semantic conventions: How would you deal with these? https://github.com/open-telemetry/opentelemetry-specification/pull/3212#issuecomment-1453934136

IMHO they belong more in this new repository than the spec, but maybe the name is then not right?

My own opinion is that semantic conventions may encompass a broader scope (all of which is currently in the spec): The attribute "data dictionary", conventions on when to use which attributes group, how to name particular spans, how to model particular scenarios in span structures, and in some cases also some "How To" / "Cookbook" / "Best-practice" instructions on how to implement particular (cross-language) instrumentations that go a bit beyond that.